CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 All notable changes to this project will be documented in this file.
 This project adheres to Semantic Versioning (http://semver.org/).
 
+### 5.5.0
+* Add GraphQL session tests
+* Move product id parsing function from CMP to CategoryMerchandisingResult
```

Comment:
Update the second point to:

Add functionality to parse the list of product ids inside CategoryMerchandisingResult